python-package/lightgbm/sklearn.py

```diff
@@ -509,7 +509,10 @@ def __init__(
         random_state: Optional[Union[int, np.random.RandomState, np.random.Generator]] = None,
         n_jobs: Optional[int] = None,
         importance_type: str = "split",
-        **kwargs: Any,
+        early_stopping: bool = False,
+        n_iter_no_change: int = 10,
+        validation_fraction: Optional[float] = 0.1,
+        **kwargs,
```

Comment:
```suggestion
        **kwargs: Any,
```

Why was this type hint removed? If it was just an accident, please put it back to reduce the size of the diff.